### PR TITLE
dumb component sheet

### PR DESCRIPTION
**Pull request by chrisnojima**
This is the initial dumb component sheet. There is a simple filter on top (driven by local-debug so you get it on refresh). For now it's all on one page so we can do stuff like screenshot it easy, we might need to make it more complex later. There are tickets in the backlog to move more things to it, i already moved tracker / most of signup

### CODE REVIEW

File: shared/more/components-tracker.js [hunk 1]
```diff
@@ -1,14 +0,0 @@
-import React, {Component} from 'react'
```
Comment (chrisnojima) [inline shared/more/components-tracker.js]: don't need this component specific sheet anymore

File: shared/more/dumb-sheet.render.desktop.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  render () {
+    const componentMap: DumbMap = {
+      ...CommonMap,
+      ...LoginMap,
+      ...SignupMap,
+      ...TrackerMap,
+      ...PinentryMap
+    }
+
+    return (
```
Comment (chrisnojima) [inline shared/more/dumb-sheet.render.desktop.js]: we could make this fancier later but for now just blast it all into one thing. easier to screenshot it this way

File: shared/more/dumb-sheet.render.desktop.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      <Box style={{flex: 1, padding: 20}}>
+        {Object.keys(componentMap).map(key => {
+          const map = componentMap[key]
+          // $FlowIssue TODO
+          const Component = map.component
```
Comment (chrisnojima) [inline shared/more/dumb-sheet.render.desktop.js]: i get a weird flow issue here

File: shared/more/dumb.js.flow [hunk 4]
```diff
@@ -0,0 +1,11 @@
+// @flow
+import {Component} from 'react'
+
+export type DumbMap = {
```
Comment (chrisnojima) [inline shared/more/dumb.js.flow]: all the dumb.desktop.js files basically export this

**Comment by chrisnojima** [comment]
@MarcoPolo  can we talk about this today

File: shared/more/dumb-sheet.render.desktop.js (hunk 3 above)
Comment (MarcoPolo) [inline shared/more/dumb-sheet.render.desktop.js]: What's the issue?

**Comment by chrisnojima** [comment]
@keybase/react-hackers

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by MarcoPolo** [comment]
:+1:

**Pull request merged**